build.sh

```diff
@@ -10,6 +10,14 @@ which dotnet 2>/dev/null || {
 set -e
 dotnet --info
 dotnet restore
+dotnet restore MoreLinq.NoConflictGenerator/MoreLinq.NoConflictGenerator.csproj
+codegen() {
+    dotnet run -p MoreLinq.NoConflictGenerator/MoreLinq.NoConflictGenerator.csproj -c Release -- $2 $3 $4 $5 $6 $7 $8 $9 > $1
```

Comment:
this can be simplified:

```shell
dest="$1"
shift
dotnet run -p MoreLinq.NoConflictGenerator/MoreLinq.NoConflictGenerator.csproj -c Release -- "$@" > "$dest"
```